README.md

```diff
@@ -10,7 +10,7 @@ blob      | Package blob provides types and functions for working with blobs, bl
 inclusion | Package inclusion contains functions to generate the blob share commitment from a given blob.
 merkle    | Package merkle computes a deterministic minimal height Merkle tree hash.
 namespace | Package namespace contains the Namespace data structure.
-shares    | Package shares contains the Share data structure.
+shares    | package share contains the Share data structure.
```

Comment:
```suggestion
share     | Package share contains the Share data structure.
```

Reply:
I should also remove the other packages that no longer exist. Thanks for pointing it out